tests/test_stream.py

```diff
@@ -1447,3 +1452,63 @@ def test_pretty_subarray(self):
                 actual = json.load(f)
 
             assert actual == expected
+
+
+class StreamsAreStringsJSONEncoder(json.encoder.JSONEncoder):
+
+    def _emit_stream(self, stream):
+        def _strip_quotes(string):
+            return string[1:-1]
+        chunksize = 4096
+        yield '"'
+        while True:
+            chunk = stream.read(chunksize)
+            if not chunk:
+                break
+            yield _strip_quotes(super(StreamsAreStringsJSONEncoder, self).encode(chunk))
+        yield '"'
+
+    def iterencode(self, value, **args):
+        if hasattr(value, 'read'):
+            return self._emit_stream(value)
+        return super(StreamsAreStringsJSONEncoder, self).iterencode(value, **args)
+
+
+class TestStringStream(object):
+
+    @pytest.fixture(autouse=True)
+    def chdir(self, tmpdir):
+        tmpdir.chdir()
+
+    def test_quotes(self):
+        with open('input', 'w') as f:
+            f.write('The "thing" thing')
+
+        s = jsonstreams.Stream(jsonstreams.Type.object, filename='foo', encoder=StreamsAreStringsJSONEncoder)
+        with open('input', 'r') as f:
+            s.write('foo', f)
+        s.close()
```

Comment:
you shouldn't need to call `close` here, that's the whole point of using `with open()`, it ensures that the resource is closed wen you leave the scope.

Reply:
I picked the wrong test case to copy from, then.  I picked `test_basic` because it was at the top.   I see now that most of the other tests use the context-manager interface.   I've switched it to the context-manager interface.  (The `with` that was there was for the input file, not the jsonstream.)  Thanks!